src/main/scala/nl/knaw/dans/easy/agreement/Configuration.scala

```diff
@@ -15,14 +15,32 @@
  */
 package nl.knaw.dans.easy.agreement
 
+import java.util.Properties
+
 import better.files.File
 import better.files.File.root
 import org.apache.commons.configuration.PropertiesConfiguration
 
 case class Configuration(version: String,
                          serverPort: Int,
-                         // other configuration properties defined in application.properties
-                        )
+                         templateResources: File,
+                        ) {
+
+  val templateDir: File = templateResources / "template"
+  val dansLogoFile: File = templateResources / "dans_logo.png"
+  val drivenByDataFile: File = templateResources / "DrivenByData.png"
+  val pdfRunScript: File = templateResources / "pdfgen.sh"
+  val licenses: Licenses = new Licenses(new PropertiesConfiguration((templateDir / "licenses" / "licenses.properties").toJava))
+
+  def velocityProperties: Properties = {
+    new Properties {
```

Comment:
It seems that `template.file.name` is not a Velocity property but just piggybacking on `velocityProperties`. If that is correct, I would consider that unclean design. 

Just a though: why not make the velocityProperties configurable, maybe putting them in their own config file? You can pass the name of that file directly to the [VelocityEngine contructor](https://velocity.apache.org/engine/1.7/apidocs/org/apache/velocity/app/VelocityEngine.html#VelocityEngine(java.lang.String)).

Reply:
You're correct that `template.file.name` is not a Velocity property. I fixed it by passing it in as a constructor argument (02cdd1c)